auto play prompt

stefan-philip opened the pull request.

st3f4np committed:
auto play prompt

jrosen081 reviewed (approved):
🎉 🦜 (party parrot)

cindy1u0 commented on frontend/src/components/AutoPrompt.tsx:
```diff
@@ -0,0 +1,26 @@
+
+const AutoPrompt : FunctionComponent<SpeechProp> = ({ prompt }) : ReactElement => {
+
+  function talk() {
```
small nitpick: i would do const talk = () => instead of talk()
1. for consistency 
2. This is an article that ive encountered before hehe https://dev.to/ugglr/react-functional-components-const-vs-function-2kj9

cindy1u0 commented on frontend/src/components/AutoPrompt.tsx:
```diff
@@ -0,0 +1,26 @@
+
+  useEffect(() => {
+    talk();
+     const interval = setInterval(talk, 30000);
```
actually from what client wants, let's change 30 seconds to 4 seconds here

st3f4np committed:
autoprompt timing updates

st3f4np committed:
timing update

4 seconds was a bit quick
perhaps we can make time as a prop for this component
